dashboard/lib/partial_registration_helper.rb

```diff
@@ -0,0 +1,32 @@
+require 'cdo/shared_cache'
+
+# Persist user attributes into the session and/or a shared cache during a multi-step
+# registration process.
+#
+# This concern assumes it will be included on User.
+module PartialRegistrationHelper
+  extend ActiveSupport::Concern
```

Comment:
Should this go in the `concerns` directory instead?

Reply:
Ooh, good call.  I'll move it.